src/engine/BMDieTwin.php

```diff
@@ -165,7 +165,7 @@ protected function skillStr() {
         $skillStr = '';
         if (count($this->skillList) > 0) {
             foreach (array_keys($this->skillList) as $skill) {
-                if (('Mood' != $skill) && 'Mad' != $skill) {
+                if (('Mood' != $skill) && 'Mad' != $skill && 'Turbo' != $skill) {
```

Comment:
Drive-by nit: balance these parens (i.e. remove them from the Mood comparison if you don't want them on the others).